src/lib/connectors/repos/billing-invoice-licences.js

```diff
@@ -34,18 +34,8 @@ const findOne = async id => {
  * @return {Promise<Object>}
  */
 const findAll = async (licenceId, page = 1, perPage = 10) => {
-  const result = await BillingInvoiceLicence
-    .collection()
-    .where({
-      licence_id: licenceId
-    })
-    .orderBy('date_created', 'DESC')
-    .fetchPage({
-      pageSize: perPage,
-      page: page,
-      withRelated
-    });
-  return paginationHelper.paginatedEnvelope(result);
+  const results = await raw.multiRow(queries.findAllByLicenceIdForSentBatches, { licenceId });
+  return paginationHelper.paginateRawQueryResults(results, page, perPage);
```

Comment:
We should ideally not be fetching all rows from the DB if possible, and relying on the DB to get a page.

The query below looks fairly simple - I think it might be possible to create the query with Bookshelf and use Bookshelf's pagination features.